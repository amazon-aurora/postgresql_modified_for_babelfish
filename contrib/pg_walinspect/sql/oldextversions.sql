-- test old extension version entry points

CREATE EXTENSION pg_walinspect WITH VERSION '1.0';

-- List what version 1.0 contains
\dx+ pg_walinspect

-- Move to new version 1.1
ALTER EXTENSION pg_walinspect UPDATE TO '1.1';

-- List what version 1.1 contains
\dx+ pg_walinspect

DROP EXTENSION pg_walinspect;
