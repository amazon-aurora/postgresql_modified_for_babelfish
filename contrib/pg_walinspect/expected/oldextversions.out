-- test old extension version entry points
CREATE EXTENSION pg_walinspect WITH VERSION '1.0';
-- List what version 1.0 contains
\dx+ pg_walinspect
           Objects in extension "pg_walinspect"
                    Object description                     
-----------------------------------------------------------
 function pg_get_wal_record_info(pg_lsn)
 function pg_get_wal_records_info(pg_lsn,pg_lsn)
 function pg_get_wal_records_info_till_end_of_wal(pg_lsn)
 function pg_get_wal_stats(pg_lsn,pg_lsn,boolean)
 function pg_get_wal_stats_till_end_of_wal(pg_lsn,boolean)
(5 rows)

-- Move to new version 1.1
ALTER EXTENSION pg_walinspect UPDATE TO '1.1';
-- List what version 1.1 contains
\dx+ pg_walinspect
           Objects in extension "pg_walinspect"
                    Object description                     
-----------------------------------------------------------
 function pg_get_wal_block_info(pg_lsn,pg_lsn)
 function pg_get_wal_record_info(pg_lsn)
 function pg_get_wal_records_info(pg_lsn,pg_lsn)
 function pg_get_wal_records_info_till_end_of_wal(pg_lsn)
 function pg_get_wal_stats(pg_lsn,pg_lsn,boolean)
 function pg_get_wal_stats_till_end_of_wal(pg_lsn,boolean)
(6 rows)

DROP EXTENSION pg_walinspect;
